### ISSUE TITLE
added symbols are not displayed in the game

### ISSUE THREAD

**Issue by TimeOutXC** [UnderminersTeam/UndertaleModTool#1462]
### Describe the bug

Hi, I added characters to the font, first as a picture then all of them selected in the glyph, but for some reason they are not displayed in the game.

### Reproducing steps

1. I transferred the letters to a sprite with characters
![1](https://github.com/krzys-h/UndertaleModTool/assets/142457681/760b0b3e-6656-44bb-bbc8-120983a88de4)

2. I've added a markup
![2](https://github.com/krzys-h/UndertaleModTool/assets/142457681/58e62b64-ad00-4345-aab0-5e8f228623a6)

3. Wrote these characters into a string
![3](https://github.com/krzys-h/UndertaleModTool/assets/142457681/f94532be-95b5-4ff1-aeba-4de97ee27da3)

4. Save and start game
![4](https://github.com/krzys-h/UndertaleModTool/assets/142457681/53ab74b5-f3fc-4ff3-af54-64c52f34f561)

### Setup Details

1. latest
2. windows 11
3. [Undertale BNP](https://gamejolt.com/games/UndertaleBnP/574044)

**Comment by Jacky720** [comment]
Are you sure that's the right font? Undertale has separate fnt_main and fnt_maintext fonts of different sizes.

**Comment by TimeOutXC** [comment]
> Are you sure that's the right font? Undertale has separate fnt_main and fnt_maintext fonts of different sizes.

Yes, I added new characters to both fonts and they don't show up

**Comment by VladiStep** [comment]
It's probably not gonna work, but have you tried to press the "Update range" button at the bottom of the font editor?

**Comment by Miepee** [comment]
@TimeOutXC any info?

**Comment by Miepee** [comment]
closing due
